src/sentry/migrations/0449_pickle_to_json_authenticator.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.28 on 2023-05-18 16:37
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def _backfill_authenticator(apps, schema_editor):
+    Authenticator = apps.get_model("sentry", "Authenticator")
+
+    for obj in RangeQuerySetWrapperWithProgressBar(Authenticator.objects.all()):
+        # load pickle, save json
+        obj.save()
```

Comment:
Nit: You could use `update_fields=["config"]` just to save the specific field you care about here